interfaces/builtin/mpris.go

```diff
@@ -31,6 +31,17 @@ import (
 
 const mprisSummary = `allows operating as an MPRIS player`
 
+const mprisBaseDeclarationSlots = `
+  mpris:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-connection:
+      slot-attributes:
+        name: .+
```

Comment:
It could be, but all it is really saying is that 'name' must be present. The interface slot code verifies this for us.